grid/src/Cabana_Grid_FastFourierTransform.hpp

```diff
@@ -97,13 +95,21 @@ struct is_matching_array<
     : public std::true_type
 {
 };
+enum class FFTCommPattern : unsigned int
```

Comment:
@sr7cb hey Sanil, what kind of communication options would we expect to interface with in fftx? Trying to make sure our interface is generic enough

Reply:
Unfortunately I don't have enough familiarity with our MPI backend to provide a concrete answer. Please go ahead with your process and we can update/revisit down the road.